Tests/Resources/config/config.yml

```diff
@@ -25,7 +25,7 @@ assetic:
             filters: sass
     filters:
         sass:
-            apply_to: "\.sass$"
+            apply_to: "\\.sass$"
```

Comment:
I suggesting using single quotes instead, allowing to avoid the need for escaping